packages/@aws-cdk/aws-ecs-patterns/test/ec2/l3s.test.ts

```diff
@@ -804,6 +805,26 @@ test('errors when setting HTTPS protocol but not domain name', () => {
   }).toThrow();
 });
 
+test('errors when idleTimeout is over 4000 seconds', () => {
```

Comment:
Let's add three more test cases here: one that uses a value below 1 and the timeout, one with a valid timeout, and one that doesn't add a timeout.

Reply:
Working through this, I am able to add the first two tests without issue, but for the third, it appears that `idle_timeout.timeout_seconds` does not show up as an attribute in the CFN output. Still trying to figure out the best way to test this given that the CFN spec says it's 60 (i've also checked loadbalancers i've created and in the console it says 60)

Reply:
Take a look at the test below the one you've written. That's how we want to test it against the template instead of using `.toBeTruthy();` We know that if it's not showing up in the template, we're not actually passing that value to CloudFormation so it will default back to 60 seconds. 

It's fine for it to be undefined in the 4th case, that's what we're looking for. CloudFormation has their own internal logic that takes an undefined value and translates it to 60 seconds. From the integ test, you can see that the 120 seconds you set are showing up in the snapshots, so it looks like it is being added correctly.

Reply:
AH! Okay I was under the impression that if we didn't add idle_timeout it would still show up in the CFN.  

I'll add the other test checking for idle_time to be undefined when it's not added in a little bit and get that pushed up for ya. Really appreciate you taking the time to run through this with me!